index.js

```diff
@@ -40,10 +40,16 @@ const L2 = 64;
 // Helpers and Precomputes sections are reused between libraries
 // ## Helpers
 // ----------
-// error helper, messes-up stack trace
-const err = (m = '') => {
-    throw new Error(m);
-};
+function safeCaptureStackTrace(...args) {
+    if ('captureStackTrace' in Error && typeof Error.captureStackTrace === 'function') {
+        Error.captureStackTrace(...args);
+    }
+}
+function err(m = '') {
+    const e = new Error(m);
+    safeCaptureStackTrace(e, err);
```

Comment:
Given an `Error` `e`, this function removes the `err` function from the stack trace, allowing you to see the true location of the exception.